File: Blockweek_13.02.2023/Schauspieler.cs
using System.Runtime.ExceptionServices;

namespace block;
public class Schauspieler
{
    string Vorname;
    string Nachname;
    string Spitzname;
    string Geburtsort;
    string Geburtsdatum;
    string Nationalitaet;
    Int64 Groese;

    public Schauspieler(string vorname, string nachname)
    {
        this.Vorname = vorname;
        this.Nachname = nachname;
    }
    public Schauspieler(string vorname, string nachname, string spitzname, Int64 groese)
    {
        this.Vorname = vorname;
        this.Nachname = nachname;
        this.Spitzname = spitzname;
        this.Groese = groese;
    }
    public Schauspieler(string vorname, string nachname, string spitzname, string geburtsort, string geburtsdatum, string nationalitaet, long groese)
    {
        this.Spitzname = spitzname;
        this.Geburtsort = geburtsort;
        this.Geburtsdatum = geburtsdatum;
        this.Nationalitaet = nationalitaet;
        this.Groese = groese;
    }
    public Schauspieler()
    {
        this.Vorname = null;
        this.Geburtsort = null;
        this.Geburtsdatum = null;
        this.Nachname = null;
        this.Nationalitaet = null;
        this.Groese = 0;
    }


    public void set_Vorname(string vorname)
    {
        this.Vorname = vorname;
    }

    public void set_Nachname(string nachname)
    {
        this.Nachname = nachname;
    }

    public void set_Geburtsort(string geburtsort)
    {
        this.Geburtsort = geburtsort;
    }

    public void set_Geburtsdatum(string geburtsdatum)
    {
        this.Geburtsdatum = geburtsdatum;
    }

    public void set_Nationaliaet(string nationaliaet)
    {
        this.Nationalitaet = nationaliaet;
    }
    public void set_Groese(Int64 groese)
    {
        this.Groese = groese;
    }


    public string get_Vorname()
    {
        return this.Vorname;
    }

    public string get_Nachname()
    {
        return this.Nachname;
    }

    public string get_Geburtsort()
    {
        return this.Geburtsort;
    }

    public string get_Geburtsdatum()
    {
        return this.Geburtsdatum;
    }

    public string get_Nationaliaet()
    {
        return this.Nationalitaet;
    }
    public Int64 get_Groese()
    {
        return this.Groese;
    }

}

class Program
{
    
    static void Main()
    {
        Schauspieler schauspieler = new Schauspieler();

        schauspieler.set_Vorname("Harrison");

        Console.WriteLine(schauspieler.get_Vorname());
    }

    
}

File: Blockweek_13.02.2023/java/Schauspieler.java
public class Schauspieler
{
    private String Vorname;
    private String Nachname;
    private String Spitzname;
    private String Geburtsort;
    private String Geburtsdatum;
    private String Nationalitaet;
    private double Groesse;
    
    public Schauspieler(String vorname, String nachname) {
        this.Vorname = vorname;
        this.Nachname = nachname;
        this.Spitzname = "";
        this.Geburtsort = "";
        this.Geburtsdatum = "";
        this.Nationalitaet = "";
        this.Groesse = 0;
    }

    public Schauspieler(String vorname, String nachname, String spitzname, double groesse) {
        this.Vorname = vorname;
        this.Nachname = nachname;
        this.Spitzname = spitzname;
        this.Geburtsort = "";
        this.Geburtsdatum = "";
        this.Nationalitaet = "";
        this.Groesse = groesse;
    }

    public Schauspieler(String vorname, String nachname, String spitzname, String geburtsort, String geburtsdatum, String nationalitaet, double groesse) {
        this.Vorname = vorname;
        this.Nachname = nachname;
        this.Spitzname = spitzname;
        this.Geburtsort = geburtsort;
        this.Geburtsdatum = geburtsdatum;
        this.Nationalitaet = nationalitaet;
        this.Groesse = groesse;
    }
    
    public Schauspieler() {
        this.Vorname = "";
        this.Nachname = "";
        this.Spitzname = "";
        this.Geburtsort = "";
        this.Geburtsdatum = "";
        this.Nationalitaet = "";
        this.Groesse = 0;
    }

    public String get_Vorname() {
        return Vorname;
    }

    public void set_Vorname(String vorname) {
        Vorname = vorname;
    }

    public String get_Nachname() {
        return Nachname;
    }

    public void set_Nachname(String nachname) {
        Nachname = nachname;
    }

    public String get_Spitzname() {
        return Spitzname;
    }

    public void set_Spitzname(String spitzname) {
        Spitzname = spitzname;
    }

    public String get_Geburtsort() {
        return Geburtsort;
    }

    public void set_Geburtsort(String geburtsort) {
        Geburtsort = geburtsort;
    }

    public String get_Geburtsdatum() {
        return Geburtsdatum;
    }

    public void set_Geburtsdatum(String geburtsdatum) {
        Geburtsdatum = geburtsdatum;
    }

    public String get_Nationalitaet() {
        return Nationalitaet;
    }

    public void set_Nationalitaet(String nationalitaet) {
        Nationalitaet = nationalitaet;
    }

    public double get_Groesse() {
        return Groesse;
    }

    public void set_Groesse(double groesse) {
        Groesse = groesse;
    }
}